app/views.py

```diff
@@ -11,10 +11,16 @@
 from werkzeug.security import generate_password_hash, check_password_hash
 import jwt
 from functools import wraps
+from app.requests import Requests
+from app.user import User
+thisUser = User('', '', '', '', '')
+thisRequest = Requests('', '', '', '', '', '', '')
```

Comment:
don't instantiate your user and requests classes like this, if you need to use the instance of a class create it in the code instead of at the top of the file.